doc/source/whatsnew/v0.19.0.txt

```diff
@@ -790,6 +791,50 @@ Sparse Changes
 
 These changes allow pandas to handle sparse data with more dtypes, and for work to make a smoother experience with data handling.
 
+
+``int64`` and ``bool`` support enhancements
+"""""""""""""""""""""""""""""""""""""""""""
+
+Sparse data structure now gained enhanced support of ``int64`` and ``bool`` ``dtype`` (:issue:`667`, :issue:`13849`)
+
+Previously, sparse data have ``float64`` dtype by default, even if all inputs are ``int`` or ``bool``. You had to specify ``dtype`` explicitly to create sparse data with ``int64`` dtype. Also, you must specify ``fill_value`` to actually sparcify the data, becuase ``fill_value`` 's default is ``np.nan`` which doesn't appear in ``int64`` data.
```

Comment:
have -> were
even if all inputs are -> even if all inputs were ``int`` or ``bool`` dtype.

Unclear what the ``Also, you must specify`` sentence means....can you reword?